### DISCUSSION TITLE
M1Max Monterey

### DISCUSSION THREAD

**Discussion by 100below** [apohl79/audiogridder#798] [General]
Has anyone tested M1 native in Monterey yet? Monterey seems to have a lot of issues with plugins. If AG works in the new Mac Book Pros, this would help a lot of early adopters bridge to non working plugs.

**Comment by JamesXXV** [comment]
Well.... it doesn't work perfectly on the new M1 Pro that I can tell, it needs a lot of buffer even when both computers have a lot of processing power, i5-7th gen on the server should be enough for 256 buffer size, from what I can tell its something to do with the transferring of midi and audio 'always' but even on the other setting it still hogs a single core crashing CoreAudio. I think it can be optimized but it also like boils down to how logic handles cores and processes , while it does distribute the plugins on different tracks to different cores but the audio and midi throughput seems to fall under one single process that isn't distributed

**Reply by 100below** [reply]
Thanks for the report. Mine will be here soon and I will report back with my experiments.
